Thorium-Sim/thorium-nova
# Flight director flex layout

## Description

Provides a flex layout which can be used to arrange flight director controls.

**Merge #296 into this branch before merging this into develop**

## Related Issue








Closes #276 

## How do you know the changes work correctly?

Manual test:
Start a new flight.
Assign the client to the flight director station
Use the controls at the top to add the Login core.
Open another client, assign it to a real station, and log in to see the login core change.

## Screenshots (if appropriate):
<img width="1904" alt="Screen Shot 2022-04-23 at 10 05 16 AM" src="https://user-images.githubusercontent.com/6558157/164909337-28df7572-f02c-45d3-b7b0-03f63ba4a794.png">

## Types of changes



      change)

## Checklist:
